spec/features/staff/users_can_create_a_fund_level_activity_spec.rb

```diff
@@ -154,7 +154,6 @@
 
         click_button I18n.t("form.activity.submit")
         expect(page).to have_content "Planned start date can't be blank"
```

Comment:
Perhaps we should replace this error message with a new expectation that in this case an error appears around the `actual_start_date`?

![Screenshot 2020-06-08 at 14 29 06](https://user-images.githubusercontent.com/912473/84035979-7caa2200-a994-11ea-8273-d92fb202b598.png)

Reply:
Now I look at it, I think there's 2 additional things to consider with the new view:

1. should planned_end_date now have "(optional)" in the label?
2. how do we label the `planned_start_date` and `actual_start_date` to inform the user that it requires one or the other?

Reply:
Added the `(optional)` tag to planned end date and added a better error message (with input from Louise)